bin/impl/redeploy.sh

```diff
@@ -67,6 +67,9 @@ if [ -n "$TARBALL" ]; then
   $SED "s#HADOOP_PREFIX=/path/to/hadoop#HADOOP_PREFIX=$HADOOP_PREFIX#g" $FLUO_HOME/conf/fluo-env.sh
 
   if [ $SETUP_METRICS = "true" ]; then
-    cp $FLUO_DEV/conf/fluo/metrics.yaml $FLUO_HOME/conf/
+    echo "io.fluo.metrics.reporter.graphite.enable=true" >> $FLUO_PROPS
```

Comment:
What do you think about replacing the commented out default properties using sed rather than appending to fluo.properties?